book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rent_book(rental)
+    @rentals.push(rental)
+    rental.book = self
```

Comment:
- [ ] As mentioned in the project requirements, the `has-many` relationship between rental and books this is not the way it is supposed to be achieved.  to perform a renatl for a specific book, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 

```suggestion
  def rent_book(person, date)
    Rental.new(date, self, person)
```

Reply:
thank you so much!! so kind of you!